packages/odyssey-react-mui/src/Tabs.tsx

```diff
@@ -72,6 +78,12 @@ const Tabs = ({ ariaLabel, initialValue = "0", tabs }: TabsProps) => {
     []
   );
 
+  useEffect(() => {
+    if (value) {
+      setTabState(value);
+    }
+  }, [value]);
```

Comment:
I know this works fine because `value` is a `string`, but can we be more clear about our intent by checking if `value === undefined`? That way, if we do `string | number`, then it won't break if someone passes `0`.